### DISCUSSION TITLE
Custom keyboard shortcuts

### DISCUSSION THREAD

**Discussion by ToRRent1812** [quodlibet/quodlibet#4269] [Ideas]
Hi, I really enjoy using quod libet, it's really powerful audio player that suits my needs, however it lacks one feature that I use quite often in multimedia apps - shortcuts.
I don't like default shortcuts in quod, I also think they miss a lot of actions. I would love to increase and decrease volume, play/pause, switch to next, go to previous, switch playlist, all of these actions with shortcuts I can customize.

**Comment by jej** [comment]
Did you active "MPRIS D-Bus Support" plugin? At least for keyboard multimedia shortcuts...
